sdk/src/main/java/io/radar/sdk/RadarUtils.kt

```diff
@@ -123,16 +123,13 @@ internal object RadarUtils {
                 || Build.PRODUCT.contains("simulator");
     }
 
-    internal fun isScreenSharing(context: Context): Boolean {
-        if (Build.VERSION.SDK_INT < 17) {
-            return false
-        }
+    internal fun hasMultipleDisplays(context: Context): Boolean {
         val displayManager = DisplayManagerCompat.getInstance(context)
-        val multipleDisplays = displayManager.displays.size > 1
-
-        val sharing = RadarSettings.getSharing(context)
+        return displayManager.displays.size > 1
+    }
```

Comment:
Looks like you removed the check on version
```
    if (Build.VERSION.SDK_INT < 17) {
            return false
        }
```
which should still be necessary given: ` minSdkVersion 16`

Reply:
Good callout but don't think this is needed any more. I took a look at all the underlying logic and don't see anything that requires API level 17. Also not getting any errors/warnings from the compiler